src/lib.rs

```diff
@@ -278,6 +280,64 @@ impl From<Ulid> for u128 {
     }
 }
 
+impl From<[u8; 16]> for Ulid {
```

Comment:
Can add a `const fn Ulid::from_bytes(_: [u8; 16])` function to allow this to be used in a const context.

Reply:
Done!